west.yml

```diff
@@ -21,6 +21,9 @@ manifest:
     - name: zephyrproject-rtos
       url-base: https://github.com/zephyrproject-rtos
 
+    - name: dsemenets
```

Comment:
it is unused as far as I see
so, pls delete these lines

Reply:
It is used until tee-supplicant not merged.